hook/main.go

```diff
@@ -175,6 +199,10 @@ func main() {
 		secret = []byte(s)
 	}
 
+	ch := make(chan event, 100)
```

Comment:
Genau, buffer auf 1 und wenn man nicht in den buffer schreiben kann, einfach droppen.